distribution/docs/src/main/resources/_contents/_transformers/searchui-transformer-contents.adoc

```diff
@@ -0,0 +1,73 @@
+==== SearchUI
```

Comment:
Isn't this the _Simple_ Search UI? Now that we have three Search UIs this name is confusing.